### PR TITLE
bpo-35885: configparser indent

### PR DESCRIPTION
**Pull request by idomic**
[bpo-35885](https://bugs.python.org/issue35885): configparser indent

https://bugs.python.org/issue35885

### CODE REVIEW

**Comment by idomic** [comment]
Adding @taleinat

File: Lib/configparser.py [hunk 1]
```diff
@@ -132,10 +132,12 @@
     set(section, option, value)
         Set the given option.
 
-    write(fp, space_around_delimiters=True)
+    write(fp, space_around_delimiters=True, indent='')
         Write the configuration state in .ini format. If
         `space_around_delimiters' is True (the default), delimiters
         between keys and values are surrounded by spaces.
+
+        If `indent' is empty (the default), the keys won't be indented.
```
Comment (taleinat) [inline Lib/configparser.py:140]: This doc-string needs to mention what will happen when indent is set to a non-empty value.

File: Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+:class:`configparser` :func:`write()` method have a new parameter `indent` of type string.
```
Comment (taleinat) [inline Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst:1]:
```suggestion
Add a new parameter ``indent`` to :meth:`configparser.write()`.

```

File: Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:class:`configparser` :func:`write()` method have a new parameter `indent` of type string.
+When indent is set, the parser will indent the lines according to the given input value.
```
Comment (taleinat) [inline Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst:2]: IMO this is too much information for the NEWS entry, but something like this should be added to the doc-string.

File: Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+:class:`configparser` :func:`write()` method have a new parameter `indent` of type string.
+When indent is set, the parser will indent the lines according to the given input value.
+Contributed by Ido Michael in :issue:`35885`
```
Comment (taleinat) [inline Misc/NEWS.d/next/Library/2020-02-16-17-38-33.bpo-35885.VijyzZ.rst:3]:
```suggestion
Contributed by Ido Michael.
```

**Review by taleinat — CHANGES_REQUESTED** [review]

**Comment by taleinat** [comment]
@idomic, something with how you're managing your PR branches is very problematic. Would you mind closing this PR and opening a new, clean one?

**Pull request closed without merging**